--- ./www/sites/all/libraries/google-api-php-client/src/Google/Auth/OAuth2.php
+++ ./www/sites/all/libraries/google-api-php-client/src/Google/Auth/OAuth2.php
@@ -179,8 +179,10 @@
    * @param string $token
    * @throws Google_Auth_Exception
    */
-  public function setAccessToken($token)
+  public function setAccessToken($tokenArg)
   {
+    $token = json_decode(preg_replace("/\\\/", "", trim($tokenArg, "\"")), $decode_to_assoc_array = true);
+
     if ($token == null) {
       throw new Google_Auth_Exception('Could not json decode the token');
     }
